keps/sig-network/2681-pod-host-ip/README.md

```diff
@@ -0,0 +1,307 @@
+MY_HOST_IPS=fd00:10:20:0:3::3,10.20.3.3
+
+# PodHostIPs FeatureGate is disabled
+MY_HOST_IPS=
+```
+
+### Test Plan
+
+Test whether FeatureGate behaves as expected when it is turned on or off
+Test whether Downward API supports `status.hostIPs`
+
+#### Expected behavior
+
+- If PodHostIPs FeatureGate is enabled:
+  - `status.hostIPs` there will be all host IPs (IPv4 and IPv6)
+- Else:
+  - `status.hostIPs` will be empty and the ApiServer will reject Downward API `status.hostIPs`, 
+    if Pods already existing Downward API `status.hostIPs`, ensure to ignore it and not affect others.
+
+### Graduation Criteria
+
+#### Alpha
+
+- Feature implemented behind a feature flag
+  - Add `status.hostIPs` to the PodStatus API
+  - Downward API support for `status.hostIPs`
+- Basic units and e2e tests completed and enabled
+
+#### Alpha -> Beta Graduation
+
+- Gather feedback from developers and users
+- Expand the e2e tests with more scenarios
+- Tests are in Testgrid and linked in the KEP
+
+#### Beta -> GA Graduation
+
+- 2 examples of end users using this field
+- Allowing time for feedback
+
+#### Removing a Deprecated Flag
+
+- Announce deprecation and support policy of the existing flag
+- Two versions passed since introducing the functionality that deprecates the flag (to address version skew)
+- Address feedback on usage/changed behavior, provided on GitHub issues
+- Deprecate the flag
+
+### Upgrade / Downgrade Strategy
+
+N/A
+
+### Version Skew Strategy
+
+N/A
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name:
+  - Components depending on the feature gate:
+- [ ] Other
+  - Describe the mechanism:
+  - Will enabling / disabling the feature require downtime of the control
+    plane?
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+###### Does enabling the feature change any default behavior?
+
+No.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. Using the featuregate is the only way to enable/disable this feature.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+The feature should continue to work just fine.
+
+###### Are there any tests for feature enablement/disablement?
+
+Yes.
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+No.
+
+###### What specific metrics should inform a rollback?
+
+N/A
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+No.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No.
+
+### Monitoring Requirements
+
+N/A
```

Comment:
It's not required for Alpha, but it's definitely not N/A.

Similarly for Scalbility at least.

Reply:
@wojtek-t

I have added the content.

Reply:
Thanks - will take a deeper look early next week.